install/kubernetes/helm/istio/values.yaml

```diff
@@ -31,6 +31,17 @@ mixer:
     enabled: true
   telemetry:
     enabled: true
+    # based on measurements 50ms p50 translates to p99 of under 1s. This is ok for telemetry which is inherently async.
+    averageLatencyThreshold: 50ms
+    # disabled, logonly or enforce
+    loadsheddingMode: enforce
+  adapters:
+    stdio:
+      enabled: false
+  resources:
+    limits:
+      cpu: 4800m
```

Comment:
Please make sure this is overridden for tests, minikube will not like it.

Reply:
I'd strongly recommend not putting a limit on the CPU of 4800m, this will cause kubernetes to require the deployment for the mixer to only exist on a an 8 cpu core only, which on AWS requires a `m5.2xlarge` dedicated machine which significantly increases the cost of running istio and causes its scaling to be in a step pattern rather than linearly with traffic. Removing the limit would allow it to scale up as high as available and not put an expensive dedicated service requirement on istio.

Reply:
Yeah,  I'm going to go ahead and apologize profusely, this was my misunderstanding as when I went back and looked closer the node went un-scheduable for a completely unrelated reason.  Sorry. Sorry. Sorry.  Ignore my earlier comment.